README.md

```diff
@@ -31,16 +31,27 @@ To be able to show your App Name in the Cupertino Alert Dialog, add the followin
 
 ## Usage
 
-Just add this code inside the `initState()` life cycle method on the first home screen of your app, and it will handle the rest.
+Just add this code whenever you want to display the update alert, and it will handle the rest.
 
 ```dart
-Future.delayed(Duration.zero, () {
-  NativeUpdater.displayUpdateAlert(
-    context: context,
-    requiresUpdate: true,
-    appStoreUrl: '<Your App Store URL>',
-  );
-});
+NativeUpdater.displayUpdateAlert(
+  context: context,
+  forceUpdate: true,
+);
+```
+
+Or with the optional parameters to customize the alert.
+
+```dart
+NativeUpdater.displayUpdateAlert(
+  context: context,
+  forceUpdate: true, /// Set to true if you are forcing an update. Set to false if you are giving an option to update later.
```

Comment:
Amazing! I would remove the comments from the example and let it plain with no comments. People will try to copy the code and the comments will go together.

Move out the comments to text in the document explaining every argument is a must.

Reply:
I'm on it...

Reply:
Resolved in #5